requirements.txt

```diff
@@ -11,3 +11,4 @@ PyQt6-Qt6~=6.5.3; sys_platform == 'darwin'
 python-dateutil~=2.7
 pywin32; sys_platform == 'win32'
 PyYAML>=5.1, <7
+chardet~=5.2.0
```

Comment:
Because of the way how we use separate requirements files for specific binary builds (on master this is currently `requirements-win.txt` and `requirements-macos-11.txt`) you need to add it there as well. Please do so with a fixed version:

```
chardet==5.2.0
```

But keep it as `chardet~=5.2.0` here in the generic requirements.txt.

Reply:
BTW, there are alternatives to `chardet` like https://pypi.org/project/charset-normalizer/

Reply:
oh, didn't notice that, I will check them out and see if any one fixes some [issues](https://github.com/metabrainz/picard/pull/2372#discussion_r1521376049) we have with `chardet`